website/docs/api/span.md

```diff
@@ -506,17 +506,26 @@ sent = span.sent
 sent = doc[sent.start : max(sent.end, span.end)]
 ```
 
+## Span.sents {#sents tag="property" model="sentences"}
```

Comment:
I think something went wrong while copy/pasting. The Example and table for `span.sent` has been deleted. Could you restore this?

Reply:
Yes, sorry about it. I corrected.